packages/@ember/-internals/container/lib/registry.ts

```diff
@@ -629,6 +629,9 @@ export default class Registry implements IRegistry {
     return assign({}, fallbackKnown, localKnown, resolverKnown);
   }
 
+  fullNameErrorMessage =
+    'Angle bracket nested component requires the :: separator between the names and the directory names must be in PascalCase.';
```

Comment:
wondering, could we add correct notation for it into message? or correct example (`<Foo-Bar/Baz />` should be `<FooBar::baz />`

Reply:
I thought about it but usually an `Error` message does not have examples and just points to the source of a problem, unlike `Warning`.

Reply:
@rwjblue @mehulkar @jenweber what do you think?